src/Admin/OnBoarding/Steps/SmsGateway.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace WP_SMS\Admin\OnBoarding\Steps;
+
+use Exception;
+use WP_SMS\Admin\OnBoarding\StepAbstract;
+use WP_SMS\Components\RemoteRequest;
+use WP_SMS\Option;
+use WP_SMS\Utils\Request;
+
+class SmsGateway extends StepAbstract
+{
+    const CACHE_DURATION = 43200; // 12 hours in seconds
+
+    public function getFields()
+    {
+        return ['name'];
+    }
+
+    protected function initialize()
+    {
+        if (Request::get('step') == $this->getSlug()) {
+            $this->setData('gateways', $this->getAllPages());
+        }
+    }
+
+    private function getAllPages()
+    {
+        $gateways = array();
+        $page     = 1;
+
+        try {
+            do {
+                $request = new RemoteRequest('get', "https://staging.wp-sms-pro.com/wp-json/wp/v2/gateway?per_page=100&page={$page}");
```

Comment:
Let's change it to production. https://wp-sms-pro.com/wp-json/wp/v2/gateway @abouterf